common/api/core-frontend.api.md

```diff
@@ -3280,16 +3279,15 @@ export class FrameStatsCollector {
 
 // @public (undocumented)
 export interface FrontendHubAccess {
+    getChangesetFromNamedVersion(arg: IModelIdArg & {
+        versionName?: string;
+    }): Promise<ChangesetIndexAndId>;
     // (undocumented)
-    getChangesetIdFromNamedVersion(arg: IModelIdArg & {
-        versionName: string;
-    }): Promise<ChangesetId>;
-    // (undocumented)
-    getChangesetIdFromVersion(arg: IModelIdArg & {
+    getChangesetFromVersion(arg: IModelIdArg & {

```

Comment:
@skirby1996 not sure if you have already handled this in the codemod tool but this will need to be updated.

@aruniverse mind updating the NextVerison.md with the replacement here?

Reply:
updated, resolving to prevent pr from completing